examples/callbacks/README.md

```diff
@@ -0,0 +1,3 @@
+This example is actually several mini-examples that should how callbacks can be used.  The
+emphasis on how they can be used as a port/adapter style system with the embedding application
```

Comment:
IMO you should replace "port/adaptor style" with a few more words that describes what you mean, as those terms don't appear anywhere else in the repo best I can tell and different people often have different pictures for these generic terms for patterns.